docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/01_instrument_intro_prerequisites.md

```diff
@@ -0,0 +1,57 @@
+     - allows for more complex validation and scoring
+     - Can easily be converted back to LINST
+ - *Cons:*
+     - Requires a developer
+     - Significant maintenance required over time to adapt to new LORIS releases
+     - less stable as some elements might change get deprecated over time
+
+### How is Instrument Data Stored?
+
+The data entered by users in an instrument form can be stored in 2 different formats: **SQL table** or **JSON string**. 
+
+JSON data storage makes instrument installation easier by removing the need for table creation in the database and reducing maintenance efforts in the long term. However, SQL storage allows for data amendments to be done directly through SQL commands without the need to parse the JSON data using a tool.
+
+The JSON data string is stored in the `Data` column of the `flag` table. If the SQL mode is enabled, a new table must be created and data is stored into the dedicated instrument table.
+
+>_**Note:** The support for JSON data storage is a new implementation that is meant to deprecate the use of SQL tables in the long term but is currently offered as an alternative only._
+
+## Prerequisites
+
+1. Prepare your **Case Report Forms (CRFs)** including any “lookup” tables used for scoring/normalization and any documentation defining possible relationships between fields for validation.
+2. Determine which **sections of the questionnaire should be included or omitted** from the online version. Typically, unless they pertain to the patients themselves, administrative details included on the original form can be ignored, but instructions that will be helpful for the examiner/patient might be included. 
+3. Decide **how all your fields would map to the form element** types allowed by LORIS
+4. Determine if your instrument requires **special certifications** to be administered. If so, only certified examiners will be able to sign the instrument. (*For more information, see [Additional Configurations - Certification](04_instrument_additional_configurations.md#Certification) section*)
+5. Determine if your instrument will have **special access restrictions** other than LORIS users from the same site and the same project. LORIS allows for instrument specific access permissions if needed. (*For more information, see [Additional Configurations - Permissions](04_instrument_additional_configurations.md#Permissions) section*)
+6. Determine if your instrument will be **administered by an examiner** and Data-entry will be done by personnel **OR** if the instrument will be completed by the **candidate directly**.(*For more information, see [Additional Configurations - Surveys](04_instrument_additional_configurations.md#Surveys) section*)
+7. Decide which **Quality Assurance (QA)** features you would like to use for the instrument [Double Data Entry](05_instrument_quality_assurance.md#Double-Data-Entry) and/or [Validity Flagging](05_instrument_quality_assurance.md#Validity)
```

Comment:
After this line, give people a brief summary of the rest of the instrument installation process: 
- Then, make the instrument and configure the battery 
- install and test the instrument
- deploy

Reply:
but that is later in the documentation not here

Reply:
I'm re-reading this and I think you mean outside of the prerequisite section? add a new section for installation summary section ? is that it?

Reply:
Yes, that would be helpful - a roadmap to instrument installation since it's such a long/forking process